src/devtools/client/debugger/src/components/Editor/NewSourceNag.tsx

```diff
@@ -7,15 +7,20 @@ import styles from "./NewSourceNag.module.css";
 export default function NewSourceNag() {
   const [showAddBreakpointNag] = useNag(Nag.FIRST_BREAKPOINT_ADD);
   const [showEditBreakpointNag] = useNag(Nag.FIRST_BREAKPOINT_EDIT);
+  const [showConsoleNavigate] = useNag(Nag.FIRST_CONSOLE_NAVIGATE);
 
   if (!showAddBreakpointNag && !showEditBreakpointNag) {
     return null;
   }
 
   return (
-    <div className={styles.Nag}>
-      <MaterialIcon iconSize="xl">auto_awesome</MaterialIcon>
-      {showAddBreakpointNag ? "Now hover on a line number" : "Now edit your print statement"}
-    </div>
+    <>
```

Comment:
Fragment isn't needed here

Reply:
Funny thing, the page fails when I don't have it there. Can't figure out why! Marking as resolved for now.

Reply:
Suspect you still have the `{` before `!showConsoleNavigate` which is only needed in JSX-land.

This should be valid:

```
return !showConsoleNavigate ? (<...>) : null;
```

Reply:
Or, even better imo:

```
if (showConsoleNavigate) {
  return null;
}

return <...>
```

Reply:
Thanks!